jitify2.hpp

```diff
@@ -100,16 +103,19 @@
 #endif
 
 #include <algorithm>
+#include <cassert>
 #include <cctype>
 #include <climits>
 #include <cstring>
 #include <fstream>
 #include <functional>
+#include <initializer_list>
 #include <iomanip>
 #include <iostream>
 #include <list>
 #include <map>
 #include <memory>
+#include <queue>
 #include <regex>
 #include <sstream>
 #include <streambuf>
```

Comment:
```suggestion
#include <streambuf>
#include <array>
```

Incomplete type `jitify2.hpp:6549` (`std::array<value_type, Size> data_ = {};`) under visual studio 2022.